### PR TITLE
XEP-0300: Clarify textual content of the <hash/> element.

### PR DESCRIPTION
**Pull request by Flowdalic**

### CODE REVIEW

**Comment by horazont** [comment]
@tfar @Kev please comment.

**Comment by stpeter** [comment]
Seems fine. You might mention that "the Base64 output MUST NOT include whitespace and MUST set padding bits to zero" (which is what we say in XEP-0115).

File: xep-0300.xml [hunk 1]
```diff
@@ -125,7 +131,7 @@
       <td>&rfc7693;</td>
     </tr>
   </table>
-  <p>The CDATA of the &lt;hash/&gt; element MUST conform to the base64Binary datatype <note>See &lt;<link url='http://www.w3.org/TR/xmlschema-2/#base64Binary'>http://www.w3.org/TR/xmlschema-2/#base64Binary</link>&gt;.</note> and thus be encoded in accordance with Section 6.8 of &rfc2045;, which recommends that base64 data should have lines limited to at most 76 characters in length. However, any whitespace characters (e.g., '\r' and '\n') MUST be ignored.</p>
+  <p>The digest produced by the used hash algorithm is included as the XML character data of the &lt;hash/&gt; after being encoded as Base64 as specified in Section 4 of &rfc4648;. Thus the character data MUST conform to the base64Binary datatype <note>See &lt;<link url='http://www.w3.org/TR/xmlschema-2/#base64Binary'>http://www.w3.org/TR/xmlschema-2/#base64Binary</link>&gt;.</note> as defined in &w3xmlschema2;.</p>
```
Comment (stpeter) [inline xep-0300.xml]: Also, I'd say "of the &lt;hash/&gt; element"

**Comment by Flowdalic** [comment]
Thanks. I have pushed the requested changes.

**Comment by stpeter** [comment]
Great, thanks! Now we just need to get it merged and published.

**Pull request merged**